modules/exploits/linux/local/pkexec.rb

```diff
@@ -66,6 +66,20 @@ def executable_path
     @executable_path
   end
 
+  def check
+    version = cmd_exec('pkexec --version')
+    if version.nil?
+      print_error "Policykit is not installed"
+      return CheckCode::Safe
+    end
+
+    if Gem::Version.new(version.split('.')[1]) <= 0.101
```

Comment:
I have a patch for this and other issues already. Please hold while I continue testing. Thanks.